src/ethereum_test_forks/forks/forks.py

```diff
@@ -150,7 +150,7 @@ def pre_allocation(cls, block_number: int = 0, timestamp: int = 0) -> Mapping:
         return {}
 
 
-class Homestead(Frontier):
+class Homestead(Frontier, solc_name="homestead"):
```

Comment:
@marioevz I decided to explicitly specify the `solc_name` as the `solc_name()` method didn't actually return the correct name (even though it was correctly handled in `ethereum_test_tools.code.yul` with `lower()`).